srcpkgs/kgpg/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'kgpg'
+pkgname=kgpg
+version=19.08.1
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
+hostmakedepends="extra-cmake-modules"
+makedepends="kwindowsystem-devel kcrash-devel kdbusaddons-devel
+ kiconthemes-devel kjobwidgets-devel kio-devel
+ kservice-devel ktextwidgets-devel kxmlgui-devel
+ kwidgetsaddons-devel knotifications-devel kdoctools-devel
+ akonadi-contacts-devel kcontacts-devel boost-devel"
+depends="akonadi-contacts phonon-qt5-backend-gstreamer oxygen gpgme"
```

Comment:
`akonadi-contacts` is automatically detected by the soname so you can remove that here. The rest is not but seems odd to me. Why would kgpg need phonon at all and then why a specific backend? Requiring a specifc icon theme also seems odd. gpgme seems to be a transitive dependency so it will be installed automatically and does not need to be explicitely listed as dependency.

Reply:
I added oxygen because it gives it sounds. And phonon-qt5-backend-gstreamer because it fixed something at startup, I already forgot, but these are not critical dependencies (oxygen, phonon-qt5-backend-gstreamer).